### ISSUE TITLE
Update Stamen layers

### ISSUE THREAD

**Issue by ianthetechie** [osmlab/editor-layer-index#2156]
Per the [announcement from Stamen](https://maps.stamen.com/stadia-partnership/), the old Fastly CDN URLs have been shut off for a bit. We have a [migration guide](https://docs.stadiamaps.com/guides/migrating-from-stamen-map-tiles/) that covers the necessary steps to get this back up and running.

TL;DR steps:

1. Change the URL pattern to something like https://tiles.stadiamaps.com/tiles/stamen_terrain_background/{z}/{x}/{y}{r}.png (note that these tiles have ben updated and support retina/hidpi now!)
2. Update attribution / license information as noted (Stamen, Stadia Maps, and OpenMapTiles need attribution + links now).

I actually came across this repo a while ago and meant to open a PR last month, but somehow missed it so dropping an issue now to track it since it's the root cause of https://github.com/facebook/Rapid/issues/1241.

**Comment by grischard** [comment]
Would you be able to open a pull request?

**Comment by ianthetechie** [comment]
Yes! Not today but sometime next week.  I wrote the issue mostly so I wouldn’t forget 😂
